tools/monorepo-utils/src/code-freeze/commands/branch/index.ts

```diff
@@ -0,0 +1,152 @@
+/**
+ * External dependencies
+ */
+import { Command } from '@commander-js/extra-typings';
+import { parse } from 'semver';
+import { confirm } from 'promptly';
+import chalk from 'chalk';
+import ora from 'ora';
+import { setOutput } from '@actions/core';
+
+/**
+ * Internal dependencies
+ */
+import {
+	getLatestReleaseVersion,
+	doesGithubBranchExist,
+	getRefFromGithubBranch,
+	createGithubBranch,
+	deleteGithubBranch,
+} from '../../../github/repo';
+import { WPIncrement } from '../milestone/utils';
+import { Options } from './types';
+
+const getNextReleaseBranch = async ( options: Options ) => {
+	const latestReleaseVersion = await getLatestReleaseVersion( options );
+	const nextReleaseVersion = WPIncrement( latestReleaseVersion );
+	const parsedNextReleaseVersion = parse( nextReleaseVersion );
+	const nextReleaseMajorMinor = `${ parsedNextReleaseVersion.major }.${ parsedNextReleaseVersion.minor }`;
+	return `release/${ nextReleaseMajorMinor }`;
+};
+
+export const branchCommand = new Command( 'branch' )
+	.description( 'Create a new release branch' )
+	.option(
```

Comment:
What about using the `CI` or `GITHUB_ACTIONS` environment variables instead? I'd probably lean towards the `CI` variable that way it isn't GitHub Action specific, but, it might be better. Some of our other tools already behave a little bit differently when `CI` is true, so, it would be perfectly reasonable for this to as well.

Reply:
This behaviour is present in the `milestone` command as well, so if we change it here it would be good to change it everywhere in a follow up PR.